### PR TITLE
Upgrade postgreSQL image to version 13 in integration tests and make them pass

### PR DESCRIPTION
**Pull request by xbaran4**
### What does this PR do?
Upgrades postgreSQL image to version 13 in integration tests and changes the docker-maven-plugin configuration so that the its compatible with this newer image.

### What issues does this PR fix or reference?

Part of: https://github.com/eclipse/che/issues/20246 + https://issues.redhat.com/browse/CRW-2180

### How to test this PR?

`mvn clean isntall -P integration` should build successfully. Later manual testing with upgraded postgresql is required.
### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)

- [x] [The Eclipse Contributor Agreement is valid](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-eclipse-contributor-agreement-is-valid)
- [x] [Code produced is complete](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-produced-is-complete)
- [x] [Code builds without errors](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-builds-without-errors)
- [x] [Tests are covering the bugfix](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#tests-are-covering-the-bugfix)
- [x] [The repository devfile is up to date and works](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-repository-devfile-is-up-to-date-and-works)
- [x] [Sections `What issues does this PR fix or reference` and `How to test this PR` completed](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#sections-what-issues-does-this-pr-fix-or-reference-and-how-to-test-this-pr-completed)
- [x] [Relevant user documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [x] [Relevant contributing documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [x] [CI/CD changes implemented, documented and communicated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#cicd-changes-implemented-documented-and-communicated)

### Reviewers

Reviewers, please comment how you tested the PR when approving it.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -71,7 +71,7 @@
         <commons-io.version>2.8.0</commons-io.version>
         <commons-lang.version>2.6</commons-lang.version>
         <integration.mysql.db.image>quay.io/eclipse/che--centos--mysql-57-centos7:latest-e08ee4d43b7356607685b69bde6335e27cf20c020f345b6c6c59400183882764</integration.mysql.db.image>
-        <integration.postgresql.db.image>quay.io/eclipse/che--centos--postgresql-96-centos7:9.6-b681d78125361519180a6ac05242c296f8906c11eab7e207b5ca9a89b6344392</integration.postgresql.db.image>
+        <integration.postgresql.db.image>centos/postgresql-13-centos7:1</integration.postgresql.db.image>
```
Comment (nickboldt) [inline pom.xml:74]: would you prefer to use a copy of a specific build, in the `quay.io/eclipse/che--centos--postgresql...` space? 

That way you won't get unexpected updates. 

Also we likely need a downstream issue for this to switch to https://catalog.redhat.com/software/containers/rhel8/postgresql-13/5ffdbdef73a65398111b8362?container-tabs=overview&gti-tabs=registry-tokens

Comment (benoitf) [reply pom.xml:74]: I think that if we bump to a newer version, we should bump to a multi-arch image.

Else there is no urgency to move to this centos7 image

Comment (nickboldt) [reply pom.xml:74]: https://quay.io/repository/eclipse/che--centos--postgresql-13-centos7?tag=1-71b24684d64da46f960682cc4216222a7e4ed8b1a31dd5a865b3e71afdea20d2&tab=tags created but it appears to be single-arch.

Comment (nickboldt) [reply pom.xml:74]: how about we use https://quay.io/repository/eclipse/registry.redhat.io--rhel8--postgresql-13?tag=1-8755650687f2c94d7f0d08f491afa0cba97199ceb7946afed260a619a3f5f49b&tab=tags ? it's got x, arm, z, and p.

**Comment by benoitf** [comment]
@nickboldt I think you hijacked the PR replacing centos7 image by a copy from a private registry using rhel8
this image is not meant to be accessible outside an authenticated registry

**Comment by nickboldt** [comment]
>  hijacked the PR

That's a bit harsh. :( I offered a solution that met your requirement of being multiarch. 

Why can't we copy images from RHEC to quay? https://quay.io/organization/app-sre has lots of examples of images ...

**Comment by benoitf** [comment]
I think it would work if there was an ubi8-postgresql13 image on the non-authenticated registry but there is none.

> Why can't we copy images from RHEC to quay? https://quay.io/organization/app-sre has lots of examples of images ...

what are the authenticated images from registry.redhat.io on https://quay.io/organization/app-sre ?

Also it does not mean that if technically you can copy images on every registry that it's something legal or open-source compliant.

**Comment by benoitf** [comment]
image

**Comment by nickboldt** [comment]
OK, good catch. reverted my commit and removed the new quay repo.

File: pom.xml (hunk 1 above)
Comment (skabashnyuk) [reply pom.xml:74]: @xbaran4 could you update the image to `quay.io/eclipse/che--centos--postgresql-13-centos7:1-71b24684d64da46f960682cc4216222a7e4ed8b1a31dd5a865b3e71afdea20d2`

**Review by sparkoo — APPROVED** [review]

**Review by mshaposhnik — APPROVED** [review]

**Pull request merged**